bower.json

```diff
@@ -1,13 +1,15 @@
 {
-	"name": "o-ft-footer",
-	"main": ["main.scss"],
-	"ignore": [],
-	"dependencies": {
-		"o-colors": "^2.3.18",
-		"o-ft-typography": "^1.12.1",
-		"o-grid": "^2.1.5",
-		"o-assets": "^0.4.0",
-		"o-fonts": "^1.4.1",
-		"o-hoverable": ">=0.1.1 <0.3"
-	}
+  "name": "o-footer",
+  "main": [
+    "main.scss"
+  ],
+  "ignore": [],
+  "dependencies": {
+    "o-colors": "^2.3.18",
+    "o-ft-typography": "^1.12.1",
+    "o-grid": ">=3.0.0-beta.3 < 4.0.0",
+    "o-assets": "^0.4.0",
+    "o-fonts": "^1.4.1",
```

Comment:
Like with Overlay, we don't want to force a font but let products decide what font they want